doc/user/bgp.rst

```diff
@@ -148,6 +148,12 @@ bottom until one of the factors can be used.
 
    Prefer higher local preference routes to lower.
 
+   If ``bgp bestpath aigp`` is enabled, and both paths that are compared, have
```

Comment:
nit ... remove the comma between "compared" and "have"

Reply:
Fixed, thanks.